src/components/formik-forms/formik-checkbox.jsx

```diff
@@ -0,0 +1,84 @@
+const classNames = require('classnames');
+const PropTypes = require('prop-types');
+const React = require('react');
+import {Field} from 'formik';
+
+require('./formik-checkbox.scss');
+require('./formik-forms.scss');
+require('../forms/row.scss');
+
+const FormikCheckboxSubComponent = ({
+    className,
+    field,
+    id,
+    label,
+    ...props
+}) => (
+    <div className="checkbox">
+        <input
+            checked={field.value}
+            className={classNames(
+                'formik-checkbox',
+                className
+            )}
+            id={id}
+            name={field.name}
+            type="checkbox"
+            value={field.value}
+            onBlur={field.onBlur} /* eslint-disable-line react/jsx-handler-names */
+            onChange={field.onChange} /* eslint-disable-line react/jsx-handler-names */
+            {...props}
+        />
+        {label && (
+            <label
+                className={classNames(
+                    'formik-label',
+                    'formik-checkbox-label'
+                )}
+                htmlFor={id}
```

Comment:
what's this used for?

Reply:
this ties the label to the checkbox, so clicking the label checks the checkbox.